jslicenses-converse.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang=en>
+    <head>
+    </head>
+    <body>
+        <table id="jslicense-labels1">
+            <tr>
+                <td>
+                    <a href="https://cdn.conversejs.org/3.3.1/dist/converse.min.js">converse.min.js</a>
+                </td>
+                <td>
+                    <a href="https://www.mozilla.org/en-US/MPL/2.0/">MPL-2.0</a>
+                    <br />
+                    <a href="magnet:?xt=urn:btih:e95b018ef3580986a04669f1b5879592219e2a7a&dn=public-domain.txt">Public-Domain</a>
```

Comment:
`&amp;` in @href, instead of `&`.

Reply:
Changed those. It appears to be a mistake in the LibreJS documentation: https://www.gnu.org/software/librejs/manual/librejs.html#Detected-Free-Licenses

Reply:
It seems to be the format for magnet links to have &dn instead of &amp;dn - I'm changing it back just in case LibreJS doesn't detect it as a valid license.

Reply:
No you're right, my bad